### ISSUE TITLE
A tiny problem in the comment at I3D_Pytorch.py?

### ISSUE THREAD

**Issue by TianxingWu** [rimchang/kinetics-i3d-Pytorch#4]
In model/I3D_Pytorch.py line 302 to line 304:

>          nn.AvgPool3d(kernel_size=(2, 7, 7), stride=1),# (1024, 8, 1, 1)
>          nn.Dropout3d(dropout_drop_prob),
>          nn.Conv3d(1024, num_classes, kernel_size=1, stride=1, bias=True),# (400, 8, 1, 1)

it seems like the dimensions wrote in the comment should be (1024, 7, 1, 1) and (400, 7, 1, 1) since there's no padding :)
